app/templates/report/reports.html

```diff
@@ -23,19 +25,42 @@ <h1 class="text-center">FOIL Request Stats</h1>
             </div>
 
             {% if current_user.is_agency %}
-            <div id="user-filter" class="form-group">
-                {{ report_filter_form.user.label(for="agency-user-filter") }}
-                <p>This allows the graph to be filtered by the requests assigned to specific users.</p>
-                {{ report_filter_form.user(id="agency-user-filter", class="input-block-level") }}
-            </div>
+                <div id="user-filter" class="form-group">
+                    {{ report_filter_form.user.label(for="agency-user-filter") }}
+                    <p>This allows the graph to be filtered by the requests assigned to specific users.</p>
+                    {{ report_filter_form.user(id="agency-user-filter", class="input-block-level") }}
+                </div>
             {% endif %}
             <button id="submit-button" class="btn btn-success">Submit</button>
             <button id="clear-filter-button" class="btn btn-primary">Clear Filter</button>
         </div>
+        <br>
+        <hr>
+        {% if current_user.is_agency_admin %}
+            <div class="container">
+            <h2 class="text-center">Agency Reports</h2>
+            <h3>FOIL Request Acknowledgement Status</h3>
+            <form id="acknowledgment-report-form" action="/report/acknowledgment" method="POST" data-parsley-validate>
+                {{ acknowledgment_form.csrf_token }}
+                <div class="form-group">
+                    {{ acknowledgment_form.date_from.label }}
+                    {{ acknowledgment_form.date_from }}
+                </div>
+                <div class="form-group">
+                    {{ acknowledgment_form.date_to.label }}
+                    {{ acknowledgment_form.date_to }}
```

Comment:
If we are forcing the "date to" to be 1 week after the "date from" I would suggest just making this a read only input to avoid confusion. The value you enter in here won't matter because it'll be overwritten.

`{{ acknowledgment_form.date_to(readonly=true }}`